openstack/networking/v2/extensions/agents/urls.go

```diff
@@ -36,18 +37,36 @@ func dhcpNetworksURL(c *gophercloud.ServiceClient, id string) string {
 	return c.ServiceURL(resourcePath, id, dhcpNetworksResourcePath)
 }
 
+func l3RoutersURL(c *gophercloud.ServiceClient, id string) string {
+	return c.ServiceURL(resourcePath, id, l3RoutersResourcePath)
+}
+
 func listDHCPNetworksURL(c *gophercloud.ServiceClient, id string) string {
 	return dhcpNetworksURL(c, id)
 }
 
+func listL3RoutersURL(c *gophercloud.ServiceClient, id string) string {
+	// TODO
+	// hmm list should be the plain l3RoutersURL but dhcp example tell otherwise
```

Comment:
Leftover comment?

Both `listL3RoutersUR`L and `listDHCPNetworksURL` look good to me. What was the issue?

Reply:
leftover from original research